### PR TITLE
feat: add caching to services layer for local dev

### PR DESCRIPTION
**Pull request by artmsilva**

### CODE REVIEW

File: services/getNavigation.js [hunk 1]
```diff
@@ -1,17 +1,60 @@
...
+      };
+    });
+
+    const componentDocsList = componentDocs.map((doc) => {
+      const { data, content, slug, filePath } = doc;
+
+      return {
+        data: {
+          title: data?.title || "",
+          order: data?.order || null,
```
Comment (wp-aberg) [inline services/getNavigation.js:37]: When the Nav builds the status indicators are missing from the nav. Need to include status here?

Comment (artmsilva) [reply services/getNavigation.js:37]: Just pushed up a fix.

Comment (wp-aberg) [reply services/getNavigation.js:37]: yep I saw that, but it's up on line 24 for Foundations. Looks like we need one here as well.

File: services/getNavigation.js [hunk 2]
```diff
@@ -1,17 +1,61 @@
...
```
Comment (artmsilva) [inline services/getNavigation.js:37]:
```suggestion
          order: data?.order || null,
          status: data?.status || "",
```

**Pull request merged**